build/build.ps1

```diff
@@ -210,9 +218,18 @@ function Build {
 
   echo "Repo toolset used from: $RepoToolsetBuildProj"
 
-  $solution = Join-Path $RepoRoot "MSBuild.sln"
+  $createTlb = $true
```

Comment:
Don't explicitly set this to `true`; that'll try to create TLBs for projects that don't need them. Set it to false or leave it unset.

Reply:
Ok, sounds good.

Reply:
Fixed.